File: app/src/main/java/com/cloudwell/paywell/PrePSPVersion/Ui/registration_Login/airticket/flightDetails1/model/Origin.java
package com.cloudwell.paywell.PrePSPVersion.Ui.registration_Login.airticket.flightDetails1.model;

import com.google.gson.annotations.SerializedName;


@SuppressWarnings("unused")
public class Origin {

    @SerializedName("Airport")
    private Airport mAirport;
    @SerializedName("DepTime")
    private String mDepTime;

    public Airport getAirport() {
        return mAirport;
    }

    public void setAirport(Airport airport) {
        mAirport = airport;
    }

    public String getDepTime() {
        return mDepTime;
    }

    public void setDepTime(String depTime) {
        mDepTime = depTime;
    }

}

File: app/src/main/java/com/cloudwell/paywell/prepspversion/ui/ticket/airticket/airportSearch/model/Origin.kt
package com.cloudwell.paywell.prepspversion.ui.ticket.airticket.airportSearch.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class Origin : Parcelable {

    @SerializedName("Airport")
    var airport: Airport? = null
    @SerializedName("DepTime")
    var depTime: String? = null

}
